benchmark/benchmark_util/data_table_benchmark_util.cpp

```diff
@@ -156,7 +159,7 @@ void LargeDataTableBenchmarkObject::PopulateInitialTable(uint32_t num_tuples, Ra
   for (uint32_t i = 0; i < num_tuples; i++) {
     auto *const redo =
         initial_txn_->StageWrite(CatalogTestUtil::TEST_DB_OID, CatalogTestUtil::TEST_TABLE_OID, row_initializer_);
-    StorageTestUtil::PopulateRandomRow(redo->Delta(), layout_, 0.0, generator);
+    //    StorageTestUtil::PopulateRandomRow(redo->Delta(), layout_, 0.0, generator);
```

Comment:
?

Reply:
It is. I was testing if it affected the benchmark numbers at all in CI. I'm content to remove it.